spec.html

```diff
@@ -32831,7 +32831,7 @@ <h1>get %TypedArray%.prototype.byteOffset</h1>
 
       <emu-clause id="sec-%typedarray%.prototype.constructor">
         <h1>%TypedArray%.prototype.constructor</h1>
-        <p>The initial value of %TypedArray%.prototype.constructor is the %TypedArray% intrinsic object.</p>
+        <p>The initial value of %TypedArray%`.prototype.constructor` is the %TypedArray% intrinsic object.</p>
```

Comment:
why are the intrinsics not in backticks?

Reply:
The proximate cause is that, font-wise, that's generally what @allenwb did in ES6. As to the underlying cause, my guess would be that monotype/backticks indicate source code, and `%TypedArray%` isn't valid ES code.